virttest/qemu_vm.py

```diff
@@ -2043,8 +2048,8 @@ def sort_key(dev):
                     "spice_x509_key_file", "spice_x509_cacert_file",
                     "spice_x509_key_password", "spice_x509_secure",
                     "spice_x509_cacert_subj", "spice_x509_server_subj",
-                    "spice_secure_channels", "spice_image_compression",
-                    "spice_jpeg_wan_compression",
+                    "spice_secure_channels", "spice_plaintext_channels",
```

Comment:
spice_plaintext_channels is not suitable here.

"spice_keys" list holds all known params for qemu-kvm

There is no such option as "spice_plaintext_channels"